### PR TITLE
solution

### PR DESCRIPTION
**Pull request by AVyha**

### CODE REVIEW

File: db/queries.py [hunk 1]
```diff
@@ -3,44 +3,76 @@
...
 def get_delivered_or_admin_messages() -> list[Message]:
-    pass
+    return Message.objects.filter(
+        Q(user__username__startswith="admin") | Q(is_delivered=1)
+    )
 
 
 def get_count_messages_sent_by_first_name(first_name: str) -> int:
-    pass
+    return sum(
```
Comment (mltdll) [inline db/queries.py:44]: You can greatly simplify this function. You're going to need need just one query and the `.count()` method.

**Review by mltdll — CHANGES_REQUESTED** [review]

File: db/queries.py (hunk 1 above)
Comment (AVyha) [reply db/queries.py]: Thank you, its really great method.

**Review by mltdll — APPROVED** [review]
Great!

File: db/queries.py [hunk 2]
```diff
@@ -3,44 +3,68 @@
 
 
 def get_messages_that_contain_word(word: str) -> list[Message]:
-    pass
+    return [
+        message for message in Message.objects.filter(text__icontains=word)
+    ]
```
Comment (Nattalli) [inline db/queries.py:8]: I think it looks simpler, isn't it?
```python
return list(Message.objects.filter(text__icontains=word))
```

Comment (Nattalli) [reply db/queries.py:8]: the same everywhere

**Review by Nattalli — CHANGES_REQUESTED** [review]

File: db/queries.py (hunk 2 above)
Comment (AVyha) [reply db/queries.py]: agree

**Review by Nattalli — APPROVED** [review]

**Pull request closed without merging**